content/get-started/quickstart/contributing-to-projects.md

```diff
@@ -195,7 +195,7 @@ To do so, head on over to the repository on {% data variables.product.product_na
 
 ## Managing feedback
 
-Pull Requests are an area for discussion. In this case, the Octocat is very busy, and probably won't merge your changes. For other projects, don't be offended if the project owner rejects your pull request, or asks for more information on why it's been made. It may even be that the project owner chooses not to merge your pull request, and that's totally okay. Your changes exist in your fork. And who knows--maybe someone you've never met will find your changes much more valuable than the original project.
+Pull Requests are an area for discussion. In this case, the Octocat is very busy, and probably won't merge your changes. For other projects, don't be offended if the project owner chooses not to merge/rejects your pull request, or asks for more information on why it's been made, and that's totally okay. Your changes exist in your fork. And who knows--maybe someone you've never met will find your changes much more valuable than the original project.
```

Comment:
```suggestion
Pull Requests are an area for discussion. In this case, the Octocat is very busy, and probably won't merge your changes. For other projects, don't be offended if the project owner chooses not to merge your pull request, or asks for more information on why it's been made. Your changes exist in your fork. And who knows--maybe someone you've never met will find your changes much more valuable than the original project.
```

Thank you for opening this PR! ✨ 

I'm making a small suggestion to delete one (or the other) of the phrases used (since we don't tend to use "/" in the docs). 

In addition, I think the "and that's totally okay" can also be removed, given that we now have "[..] don't be offended [...]" at the start of the sentence, which is intended to convey the same sentiment. ✨